### PR TITLE
update to property files

### LINKED ISSUE #6896: Update to Citation ResourceBundle property file based on citation TSV changes
Noticed the modification for "Related Publications" Tooltip at citation.tsv file.  So I will update corresponsing ResourceBundle property file and will submit the PR soon.
Thanks

### LINKED ISSUE #6913: Spectral Resolution looks weird, odd characters
In the Astronomy metadata block, Spectral Resolution looks weird:

81743326-772dea80-946f-11ea-887a-5deedfc86c5c

See related discussion at https://github.com/GlobalDataverseCommunityConsortium/dataverse-language-packs/issues/65

### PR DESCRIPTION
**Pull request by JayanthyChengan**
- closes #6896
- closes #6913

### CODE REVIEW

File: src/main/java/propertyFiles/astrophysics.properties [hunk 1]
```diff
@@ -31,7 +31,7 @@ datasetfieldtype.astroFacility.description=The observatory or facility where the
 datasetfieldtype.astroInstrument.description=The instrument used to collect the data.
 datasetfieldtype.astroObject.description=Astronomical Objects represented in the data (Given as SIMBAD recognizable names preferred).
 datasetfieldtype.resolution.Spatial.description=The spatial (angular) resolution that is typical of the observations, in decimal degrees.
-datasetfieldtype.resolution.Spectral.description=The spectral resolution that is typical of the observations, given as the ratio λ/Δλ.
+datasetfieldtype.resolution.Spectral.description=The spectral resolution that is typical of the observations, given as the ratio \u03bb/\u0394\u03bb.
```
Comment (pdurbin) [inline src/main/java/propertyFiles/astrophysics.properties:34]: This λ/Δλ stuff seems unrelated to #6896. @JayanthyChengan do you mean for this to be in the pull request?

Comment (JayanthyChengan) [reply src/main/java/propertyFiles/astrophysics.properties:34]: It's related to https://github.com/GlobalDataverseCommunityConsortium/dataverse-language-packs/issues/65.  Thought of adding to this PR.

Comment (pdurbin) [reply src/main/java/propertyFiles/astrophysics.properties:34]: @JayanthyChengan thanks. @jggautier and I took a look at the demo server and you're right, this absolutely should be fixed! I'm putting a screenshot below for QA. Should it be fixed in astrophysics.tsv as well?

Screen Shot 2020-05-12 at 2 07 34 PM

File: src/main/java/propertyFiles/citation.properties [hunk 2]
```diff
@@ -106,7 +106,7 @@ datasetfieldtype.topicClassification.description=The classification field indica
 datasetfieldtype.topicClassValue.description=Topic or Subject term that is relevant to this Dataset.
 datasetfieldtype.topicClassVocab.description=Provided for specification of the controlled vocabulary in use, e.g., LCSH, MeSH, etc.
 datasetfieldtype.topicClassVocabURI.description=Specifies the URL location for the full controlled vocabulary.
-datasetfieldtype.publication.description=Publications that use the data from this Dataset.
+datasetfieldtype.publication.description=Publications that use the data from this Dataset. The full list of Related Publications will be displayed on the metadata tab.
```
Comment (pdurbin) [inline src/main/java/propertyFiles/citation.properties:109]: @JayanthyChengan this looks fine but do you have any insight into the fact that the longer version appears above  while the shorter version appears below? Is this by design? Here's a screenshot:

Screen Shot 2020-05-12 at 4 46 54 PM

Comment (JayanthyChengan) [reply src/main/java/propertyFiles/citation.properties:109]: @pdurbin :  corrected both issues and pushed the code.  Please verify

Comment (pdurbin) [reply src/main/java/propertyFiles/citation.properties:109]: @JayanthyChengan this looks good but there should be a release note. Please take a look at https://github.com/scholarsportal/dataverse/pull/75 and merge it if you think it looks ok.

Comment (JayanthyChengan) [reply src/main/java/propertyFiles/citation.properties:109]: @pdurbin - I merged the release note.

Comment (pdurbin) [reply src/main/java/propertyFiles/citation.properties:109]: @JayanthyChengan thanks! From your experience, do you think it's necessary to maintain "description" in both the TSV files and the properties files? I hope not because this seems like a maintenance burden. If so I think we should try to move completely to the properties files.

Comment (JayanthyChengan) [reply src/main/java/propertyFiles/citation.properties:109]: @pdurbin - 
Yes, I agree with you.  Along with "description",  I also see "watermark" which can be moved to property files and stays in one place to maintain.  

We have to review the code to verify it's not affecting any other features.  Since I am engaged with other projects, I won't be able to take this for now.  Sorry.

Comment (pdurbin) [reply src/main/java/propertyFiles/citation.properties:109]: @JayanthyChengan no problem. Thanks for your response and this pull request! Have a great weekend!

**Comment by kcondon** [comment]
hi @JayanthyChengan, would you refresh this from develop since I cannot build your branch due to a failing test that has been fixed? Thanks, Kevin

**Comment by JayanthyChengan** [comment]
@kcondon - merged latest changes from develop branch.  Please verify.

**Comment by kcondon** [comment]
@JayanthyChengan  Thanks! All set.
